api/files/serializers.py

```diff
@@ -88,6 +111,7 @@ class FileSerializer(JSONAPISerializer):
         'size',
         'provider',
         'last_touched',
+        'tags',
```

Comment:
He had added the indices, then it broke all kinds of things and he had to remove them. There's details in the PR somewhere.